sdk/eventhub/azure-eventhub/CHANGELOG.md

```diff
@@ -1,10 +1,12 @@
 # Release History
 
-## 5.0.1 (Unreleased)
+## 5.0.1 (2020-03-09)
 
 **Bug fixes**
 
-- Fixed a bug that `azure.eventhub.EventHubConsumerClient.receive()` doesn't call error handler callback on_error #9660
+- Fixed a bug that swallowed errors when receiving events with the sync `azure.eventhub.EventHubConsumerClient`  #9660
```

Comment:
we can remove the `the sync` in the sentence, `azure.eventhub.EventHubConsumerClient` implies it's the sync client.